src/common/NoResults/styled.js

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components";
+import { Header, Wrapper } from "../Loader/styled";
+import { ReactComponent as Image } from "./NoResultsPicture.svg"
+
+export const NoResultsWrapper = styled(Wrapper)``;
+
+export const NoResultsHeader = styled(Header)`
+    margin: 56px 56px 40px;
+
+    @media ( max-width: ${({ theme }) => theme.breakpoints.medium}px) {
+        margin: 36px 36px 40px;
+    };
+    
+    @media ( max-width: ${({ theme }) => theme.breakpoints.small}px) {
+        margin: 0;
+    };
+`;
+
+export const NoResultsImage = styled(Image)`
+    margin: 0 auto;
+    width: 668px;
+    height: 508.63px;
```

Comment:
These lines seem to be unnecessary. The image is centered anyway because of flex in NoResultWrapper and sizes are default. The SVG image looks exactly the same without them. Or is it needed for some other devices/browsers to work properly?